server/src/bootstraper.ts

```diff
@@ -4,6 +4,9 @@ import { AppModule } from './app.module';
 import { TYPES } from './types';
 import IConfiguration from './interfaces/IConfiguration';
 import IBootstraper from './interfaces/IBootstraper';
+import { SwaggerModule, DocumentBuilder } from '@nestjs/swagger';
+
+const packageJson = require('../package.json'); //tslint:disable-line
```

Comment:
Use imports.
You can add ""resolveJsonModule": true," into tsconfig to enable json importing.